engines/director/lingo/lingo.cpp

```diff
@@ -578,9 +578,9 @@ const char *Datum::type2str(bool isk) {
 	}
 }
 
-int Datum::compareTo(Datum d) {
+int Datum::compareTo(Datum d, bool ignoreCase) {
 	if (type == STRING && d.type == STRING) {
-		return u.s->compareTo(*d.u.s);
+		return ignoreCase ? u.s->compareToIgnoreCase(*d.u.s) : u.s->compareTo(*d.u.s);
```

Comment:
This should use `toLowercaseMac` as in #2204

Reply:
Fixed, thanks.